pkg/swagger/docsHandler.go

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2023 Greenbone AG <https://greenbone.net>
```

Comment:
Some of the license comments are `2024 Greenbone AG`
I guess this was an oversight.

Reply:
Yeah, hard to say. This file is from 2024, but a one to one copy from a file in another repo, which was created in 2024. Changed to 2024 though.